recipe/meta.yaml

```diff
@@ -73,39 +78,31 @@ requirements:
     - libboost-devel
     - pyyaml
     - ply
+    - libpisp        # [variant == "rpi_fork"]
+    # libpisp build dependency
+    - nlohmann_json  # [variant == "rpi_fork"]
   run:
     - python
 test:
   commands:
+    - cam -l
+    - qcam -h
```

Comment:
your aarch builds (all of them) are now going to fail since I'm testing the existance of `qcam`.

Whatever you decide, you should make things CONSISTENT.

Either you should 
* fix qcam for aarch + cross compilation
* Use emulation to build (this will likely work)
* disable qcam for all builds

consistency is how you get things to work across platforms.

When the time comes and you will want qcam, you can enable it again, on all platforms with a fresh mind.